cmd/network/evpn-logical-brige.go

```diff
@@ -160,9 +161,10 @@ func ListLogicalBridges() *cobra.Command {
 					log.Fatalf("Failed to get items: %v", err)
 				}
 				// Process the server response
-				for _, item := range resp.LogicalBridges {
-					log.Printf(" Created Logical Bridge \n name: %s\n vlan: %d\n vni: %d\n status: %s\n VtepIpPrefix:%s", item.GetName(), item.GetSpec().GetVlanId(),
-						item.GetSpec().GetVni(), item.GetStatus(), item.GetSpec().GetVtepIpPrefix())
+				log.Println("List Logical Bridges:")
+				for _, lb := range resp.LogicalBridges {
+					log.Println("Logical Bridge with ")
```

Comment:
In the file above you used `:` in the end of the string
Let's try to be consistent. Pls check all the cases